wsgi.py

```diff
@@ -50,6 +51,15 @@ def post_adviser_result():  # Ignore PyDocStyleBear
     adapter = AdvisersResultsStore()
     adapter.connect()
     document_id = adapter.store_document(request.json)
+    if(request.form.get("origin")):
+        url = request.form.get("origin")
+        service = url.split("/")[2].split(".")[0]
+        OpenShift.schedule_kebechet(
```

Comment:
You will need to install result-api with "openshift" extras and add Pipfile, Pipfile.lock to this pull request:

```console
pipenv install 'thoth-common[openshift]'
```